### DISCUSSION TITLE
Stop auto selecting the "For you" tab

### DISCUSSION THREAD

**Discussion by SleeplessByte** [community/community#13885] [Other Feature Feedback, Questions, & Ideas]
The new feed is absolutely horrible, as you can see from feedback in far more constructive form in this discussions board. The **great** thing about it was that it was hidden in a tab, that has the misnomer "for you" (none of the information is actually interesting _to me_). 

However, now it also auto selects the second tab. A tab that is inaccessible as it doesn't show when the component is focused. Each time I go back to github.com or refresh the page, it auto selects the tab I don't want. There is no setting to fix this behaviour.

**Comment by transcental** [comment]
This doesn't happen for me. I think it might be your browser.

**Reply by SleeplessByte** [reply]
It has nothing to do with my browser. Multiple times today this happened for multiple people. It seems either an A/B test or a deploy with some faulty code.

Example: https://github.com/github/feedback/discussions/13889

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
